logic/atrium-logic-common/src/main/kotlin/ch/tutteli/atrium/logic/IteratorAssertions.kt

```diff
@@ -9,4 +9,6 @@ import ch.tutteli.atrium.creating.AssertionContainer
 interface IteratorAssertions {
     fun <E, T : Iterator<E>> hasNext(container: AssertionContainer<T>): Assertion
     fun <E, T : Iterator<E>> hasNotNext(container: AssertionContainer<T>): Assertion
+
+    fun <E, T : Iterator<E>> containsNoDuplicates(container: AssertionContainer<T>): Assertion
```

Comment:
I just realised, that this is actually the wrong class. It should be an assertion function for Iterable,  i.e. for IterableLike

Reply:
moved